python/google/protobuf/internal/well_known_types.py

```diff
@@ -143,6 +143,9 @@ def FromJsonString(self, value):
     Raises:
       ValueError: On parsing problems.
     """
+    if not isinstance(value, str):
```

Comment:
Looks like python 2.7 is failing. Though python 2.7 has reached end of life, but protobuf haven't drop it completely. Can use six.string_types instead of str